test/integration/testdata/tabletests/interfaces/interfaces.go

```diff
@@ -0,0 +1,90 @@
+package interfaces
+
+import (
+	"context"
+	"io"
+
+	"github.com/Makpoc/gomigen/test/integration/testdata/tabletests/interfaces/inner"
```

Comment:
Again, try dot imports and packages that wrongly declare themselves.
Furthermore, you may want to try overlapping interfaces (i.e. composition of two interfaces that share the same method).

Reply:
Added `ComposeOverlappingInterfaces` integration test